src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.trim().split(';');
+
+  const stylesObj = {};
+
+  declarations.forEach((declaration) => {
+    const parts = declaration.split(':');
```

Comment:
The `split(':')` method will not handle cases where colons are part of the value, such as in URLs. Consider using a more robust parsing method if such cases need to be supported.